sdk/template/ci.yml

```diff
@@ -4,6 +4,7 @@
 resources:
   repositories:
     - repository: azure-sdk-build-tools
+      ref: refs/heads/feature-releaseVerify
```

Comment:
Remember to swap this back to master before you merge.

Reply:
@mitchdenny unfortunately this isn't the actual PR, this was me testing the automatic PR creation code which ended up creating the PR for everything I had open in that branch. This isn't the PR I'm going to be checking in.